README.md

```diff
@@ -218,6 +224,30 @@ For example, to scan a  `git` repository, start with
 $ trufflehog git https://github.com/trufflesecurity/trufflehog.git
 ```
 
+## S3 
+
+The S3 source now supports assuming IAM roles for scanning in addition to IAM users. This makes it easier for users to scan multiple AWS accounts without needing to rely on hardcoded credentials for each account.
```

Comment:
**Nit:** Remove "now" in the first sentence since the README is more of a long-lived doc.